prompt_engine/core/pipeline.py

```
@@ -123,4 +149,49 @@ def run_simple(self, user_input: str, platform: str = "Blog") -> Dict[str, Any]:
+        try:
+            # Validate platform
+            platform = self._validate_platform(platform)
+            
+            # Set up context analysis
+            context_analysis = self._setup_context(user_input, platform)
+            
+            yield f"Analyzing context for {platform}...\n"
+            
+            # Step 2: Extract intent
+            intent = self.interpreter.interpret(user_input, context_analysis)
+            yield f"Extracting intent...\n"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)